pb/lmsysmd/data_ingestion/v1/model.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package lmsysmd.data_ingestion.v1;
+
+import "buf/validate/validate.proto";
+import "google/type/date.proto";
+
+service ModelService {
+  rpc CreateModel(CreateModelRequest) returns (CreateModelResponse) {
+    option idempotency_level = NO_SIDE_EFFECTS; 
+  }
+}
+
+message CreateModelRequest {
+  Id model_id = 1 [ (buf.validate.field).required = true ];
+  Model model = 2 [ (buf.validate.field).required = true ];
+}
+
+message CreateModelResponse {
```

Comment:
Do you really need to return anything?

Reply:
You're right I forgot we talked about this already my bad.